app/controllers/general.php

```diff
@@ -418,6 +423,13 @@ function router(App $utopia, Database $dbForPlatform, callable $getProjectDB, Sw
                 requestTimeout: 30
             );
 
+            // Branded 404 override
+            if ($executionResponse['statusCode'] === 404 && $resource->getAttribute('adapter', '') === 'static') {
```

Comment:
When do we use this file vs the user own 404 file?

Reply:
We use ours only if site is static, and site is not SPA